### ISSUE TITLE
inconsistent client-side URL-routing

### ISSUE THREAD

**Issue by indirectlylit** [learningequality/kolibri#2157]
Minor issue with no user-facing impact. Some apps use URLs that look like:

```
/coach/#/6e54aec31b1a5672ec663c5e11b354c3/groups
```

Others use URLs that look like:

```
/user#/profile
```

Note that there is no `/` between `user` and `#`. This is also the case for management app URLs - both `device` and `facility`.

Recommend we standardize on URLs that have a `/` before the `#`.

**Comment by indirectlylit** [comment]
see also: #2139

**Comment by christianmemije** [comment]
Fixed in #2545
